### PR TITLE
Added debuggability to Python runtime

### PR DESCRIPTION
**Pull request by dorfire**

### CODE REVIEW

File: src/startupscriptgenerator/src/python/scriptgenerator.go [hunk 1]
```diff
@@ -265,6 +233,38 @@ func (gen *PythonStartupScriptGenerator) getCommandFromModule(module string, app
...
+	return true
+}
+
+func (gen *PythonStartupScriptGenerator) buildPtvsdCommandForModule(module string, appDir string) string {
+	waitarg := ""
+	if gen.DebugWait {
+		waitarg = " --wait"
+	}
+
+	pycmd := "python -m ptvsd --host " + DefaultHost + " --port " + gen.DebugPort + waitarg + " -m " + module
```
Comment (kichalla) [inline src/startupscriptgenerator/src/python/scriptgenerator.go]: We should start using `fmt.Sprintf` which improves readability.

File: src/startupscriptgenerator/src/python/main.go [hunk 2]
```diff
@@ -14,34 +14,63 @@ func main() {
...
 	appPathPtr := flag.String("appPath", ".", "The path to the application folder, e.g. '/home/site/wwwroot/'.")
+	
 	manifestDirPtr := common.ManifestDirFlag
-	userStartupCommandPtr := flag.String("userStartupCommand", "", "[Optional] Command that will be executed to start the application up.")
-	defaultAppFilePathPtr := flag.String("defaultApp", "", "[Optional] Path to a default file that will be executed if the entrypoint is not found. Ex: '/opt/defaultsite'")
-	defaultAppModulePtr := flag.String("defaultAppModule", "application:app", "Module of the default application, e.g. 'application:app'.")
-	virtualEnvironmentNamePtr := flag.String("virtualEnvName", "", "Name of the virtual environment for the app")
-	packagesFolderPtr := flag.String("packagedir", "", "Directory where the python packages were installed, if no virtual environment was used.")
+	
+	userStartupCommandPtr := flag.String("userStartupCommand", "", "[Optional] Command that will be executed " +
```
Comment (kichalla) [inline src/startupscriptgenerator/src/python/main.go]: 👍

File: src/startupscriptgenerator/src/python/scriptgenerator.go [hunk 3]
```diff
@@ -9,85 +9,103 @@ import (
...
-	VirtualEnvironmentName   string
-	PackageDirectory         string
-	SkipVirtualEnvExtraction bool
-	Manifest                 common.BuildManifest
+	AppPath                     string
+	UserStartupCommand          string
+	DefaultAppPath              string
+	DefaultAppModule            string
+	DefaultAppDebugModule       string
+	DebugAdapter                string // Remote debugger adapter to use. Currently, only `ptvsd` is supported.
```
Comment (gpcastro) [inline src/startupscriptgenerator/src/python/scriptgenerator.go]: we can just remove `DebugAdapter` since only one is supported

Comment (dorfire) [reply src/startupscriptgenerator/src/python/scriptgenerator.go]: I had future-proofing in mind - we might want to support PyCharm for example

**Pull request merged**